lib/msal-core/src/Storage.ts

```diff
@@ -139,20 +139,20 @@ export class Storage {// Singleton
     }
 
     setItemCookie(cName: string, cValue: string, expires?: number): void {
-        var cookieStr = cName + "=" + cValue + ";";
+        let cookieStr = cName + "=" + cValue + ";";
         if (expires) {
-            var expireTime = this.setExpirationCookie(expires);
+            const expireTime = this.setExpirationCookie(expires);
             cookieStr += "expires=" + expireTime + ";";
         }
 
         document.cookie = cookieStr;
     }
 
     getItemCookie(cName: string): string {
-        var name = cName + "=";
-        var ca = document.cookie.split(";");
-        for (var i = 0; i < ca.length; i++) {
-            var c = ca[i];
+        const name = cName + "=";
+        const ca = document.cookie.split(";");
+        for (let i = 0; i < ca.length; i++) {
+            let c = ca[i];
             while (c.charAt(0) === " ") {
```

Comment:
this seems like it could end in an endless loop possibly

Reply:
Lets pick this as and when Storage.ts is touched.